src/idfxt/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+set(SRC 
+  JSONDataInterface.cpp
+  JSONDataInterface.h
+  idd-full.h
```

Comment:
More importantly how will these be maintained?  Is this where developers will add schema changes?